src/PKSim.Core/Model/IndividualSimulation.cs

```diff
@@ -78,5 +71,23 @@ public override DataColumn PeripheralVenousBloodColumn(string compoundName)
       public override DataColumn VenousBloodColumn(string compoundName) => DataRepository.VenousBloodColumn(compoundName);
 
       public override DataColumn FabsOral(string compoundName) => DataRepository.FabsOral(compoundName);
+
+      public void ClearRatioPKCache()
+      {
+         AucDDI.Clear();
+         CMaxDDI.Clear();
+         AucIV.Clear();
+      }
+
+      public IndividualSimulation()
+      {
+         AucDDI = new Cache<string, double?>(onMissingKey: x => null);
+         AucIV = new Cache<string, double?>(onMissingKey: x => null);
+         CMaxDDI = new Cache<string, double?>(onMissingKey: x => null);
+      }
+
+      public Cache<string, double?> AucDDI { get; }
```

Comment:
Initalize when you declare so that you don't nee the constructor

 public Cache<string, double?> AucDDI { get; } = new Cache<string, double?>(onMissingKey: x => null);

Reply:
but moves those guys up. Unexpected to see member declared 1/2 way through the class